tools/llm_bench/requirements.txt

```diff
@@ -1,5 +1,6 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
 numpy
+--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/pre-release
 --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
```

Comment:
adding both nightly and pre-release extras does not lead to conflict? Which one will be used by default (first found or last or it depends from platform)?

Reply:
> adding both nightly and pre-release extras does not lead to conflict?

according to our experiments for GenAI samples - no.
Pip compares packages by version and RC is more "important" than "dev" versions 

> Which one will be used by default (first found or last or it depends from platform)?

Extra URL indexes are not prioritized somehow and all of them are expedited.